tcp.go

```diff
@@ -103,20 +122,13 @@ func (fm *Frontman) runTCPCheck(addr *net.TCPAddr, hostname string, service stri
 		err = checkHTTPS(conn, hostname, secToDuration(fm.NetTCPTimeout))
 		break
 	case "tcp":
+		// TODO: Nothing to do in tcp case or report an error? If nothing why?
```

Comment:
Because we just need to test the TCP connection on port in this case

Reply:
So a comment would be nice in this case